src/sentry/similarity/features.py

```diff
@@ -136,58 +32,25 @@ def __init__(self, function):
         self.function = function
 
     def extract(self, event):
-        try:
-            exceptions = event.data['sentry.interfaces.Exception']['values']
-        except KeyError as error:
-            logger.info(
-                'Could not extract characteristic(s) from %r due to error: %r',
-                event,
-                error,
-                exc_info=True)
-            return
-
-        for exception in exceptions:
-            try:
-                yield self.function(exception)
-            except InsufficientContext as error:
-                logger.debug(
-                    'Could not extract characteristic(s) from exception in %r due to expected error: %r',
-                    event,
-                    error)
-            except Exception as error:
-                logger.exception(
-                    'Could not extract characteristic(s) from exception in %r due to error: %r',
-                    event,
-                    error)
+        return self.function(
+            event.interfaces['sentry.interfaces.Exception'].values[0],
```

Comment:
I'm not sure this is the smartest idea, but haven't really put thought into alternatives yet.